samples/client/petstore/java-helidon-client/mp/src/main/java/org/openapitools/client/model/Cat.java

```diff
@@ -21,6 +21,8 @@
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import com.fasterxml.jackson.annotation.JsonValue;
 import org.openapitools.client.model.Animal;
+import org.openapitools.client.model.Cat;
```

Comment:
Seems weird to import itself, but separate problem

Reply:
Yeah, that's weird. I've seen that some subclasses have the `@JsonSubTypes` annotation as well (samples/client/petstore/java/feign/src/main/java/org/openapitools/client/model/Cat.java for example), which I don't think is necessary (not because of this change, the `master` branch shows this behaviour as well, but it wasn't in the samples before because the source schema didn't use discriminator values). Because of this annotation, every subclass refers to every other subclass. This may be the reason for the unnecessary import here.

Reply:
agreed it's something we can deal with later.